docs/Codebase.md

```diff
@@ -0,0 +1,93 @@
+## <a href="#codebase">Codebase</a>
+InsertionsClient solution consists of 2 projects for the application logic and 2 projects for their respective tests.
+
+Besides the solution, the documentation and the `.yml` file for the build pipeline also reside in the repo at https://github.com/dotnet/insertions-client.
+
+This section covers the highlights of both `Core` and the `Console` projects as well as the `.yml` file for the build pipeline.
```

Comment:
consider at least mentioned why the test projects are not covered.

Reply:
Done.